Description: Allow Routes == 2.0
 The issue with Routes 2.0 is only the PyPi release, though the Debian
 package is perfectly working. This patch is allowing it to be used.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-09-30

Index: glance/requirements.txt
===================================================================
--- glance.orig/requirements.txt	2014-12-23 21:41:37.000000000 +0800
+++ glance/requirements.txt	2014-12-23 21:50:35.000000000 +0800
@@ -14,7 +14,7 @@
 anyjson>=0.3.3
 eventlet>=0.15.2
 PasteDeploy>=1.5.0
-Routes>=1.12.3,!=2.0
+Routes>=1.12.3
 WebOb>=1.2.3
 argparse
 boto>=2.32.1
